docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -173,6 +173,195 @@ It is also possible to configure the scope values that would be used via the sco
 
 NOTE: If both the deployment and subsystem configuration is present, then the subsystem configuration will override the deployment values.
 
+== Sending Request Objects as a JWT
+
+NOTE: Since this attribute is a preview level feature, the WildFly server's stability level must be set to `preview`
+or lower to enable the ability to add `request` or `request_uri` to authentication requests. The stability level must
+not be changed to community or higher after configuring the attributes related to authentication request format. For more
+information about the stability levels of the WildFly server, please refer to the
+link:Admin_Guide{outfilesuffix}[Admin Guide], particularly the
+link:Admin_Guide{outfilesuffix}#Feature_stability_levels[Feature stability levels] section.
+
+According to the https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests[OpenID Connect specifications],
```

Comment:
Might be good to add an intro sentence about how it's possible to send request objects as a JWT when securing apps with OIDC.

Reply:
In general might be good to reword this a bit so that it's clear to the user what they can configure (as opposed to what is defined in the spec).